layouts/shortcodes/ksonnet-min-version.html

```diff
@@ -1 +1 @@
-<a href="https://ksonnet.io/#get-started">0.13.1</a>
\ No newline at end of file
+<a href="https://ksonnet.io/get-started/">0.13.1</a>
```

Comment:
Thanks, that's an interesting comment. In the docs we use this variable under the assumption that it will link to the ksonnet installation instructions, so I'll leave it as it is for this CL. If you think it's worth changing, we'll need to provide the link to both the version info and the installation guide, in every place in the docs where this variable is used.